src/workgroups.ts

```diff
@@ -0,0 +1,88 @@
+export type MapOfLists = { [key: string]: string[] }
+export type ListOfText = string[]
+export type ListOfLinks = string[]
+export type PlainText = string
+
+export type WorkGroup = {
+  id: string
+  type?: PlainText
+  links: ListOfLinks
+  sponsor: PlainText
+  managers: ListOfText
+  members: MapOfLists
+}
+
+export type WorkGroupMap = Map<string, WorkGroup>
+export type WorkGroupSet = WorkGroup[]
+
+export enum DisplayMode {
+  PlainText,
+  ListOfText,
+  ListOfLinks,
+  MapOfLists,
+  DAWGLink
+}
+/* A map of WorkGroup properties to intended display modes
+  Creating this assignment here prevents the need for inferring it later
+  which can be difficult for things such as PlainText vs DAWGLink which
+  are both of type `string`
+  */
+export const WorkGroupDisplayModes: Map<keyof WorkGroup, DisplayMode> = new Map([
+  ['id', DisplayMode.DAWGLink],
+  ['type', DisplayMode.PlainText],
+  ['links', DisplayMode.ListOfLinks],
+  ['sponsor', DisplayMode.PlainText],
+  ['managers', DisplayMode.ListOfText],
+  // ['subgroups', DisplayMode.ListOfText],
+  // ['members_list', DisplayMode.ListOfText],
+  ['members', DisplayMode.MapOfLists]
+])
+
+export const getFieldDisplayMode = (f: keyof WorkGroup, w: WorkGroup): DisplayMode => {
+  return DisplayMode.PlainText
+}
+
+const DefaultWorkGroupIDs = [
```

Comment:
It looks like this isn't used anywhere (we want to hide these groups by default).

Reply:
fixed, this was a temporary regression as I moved to the mapped groups in the table and has been restored